stable/sonatype-nexus/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: sonatype-nexus
-version: 1.0.1
+version: 1.0.2
```

Comment:
Please bump the chart version to 1.0.3

Reply:
Right, of course I forgot I had another PR XD